awot/io/read_hiaper_radar.py

```diff
@@ -0,0 +1,214 @@
+"""
+    awot.io.read_haiper_radar
+    =================
+    
+    Scripts to read NCAR HIAPER Cloud Radar NetCDF data files.
+    
+    https://www.eol.ucar.edu/observing_facilities/hiaper-gulfstream-gv
+    
+    Testing was done from data file provided by Nguy.
```

Comment:
We should credit Pei Tsai for the file, she's the engineer with the project at NCAR.